tmp/lanelet2_extension/include/lanelet2_extension/regulatory_elements/autoware_traffic_light.hpp

```diff
@@ -27,17 +27,17 @@
 
 namespace lanelet::autoware
 {
-struct AutowareRoleNameString
-{
-  static constexpr const char LightBulbs[] = "light_bulbs";
-};
-
 class AutowareTrafficLight : public lanelet::TrafficLight
 {
 public:
   using Ptr = std::shared_ptr<AutowareTrafficLight>;
   static constexpr char RuleName[] = "traffic_light";
 
+  struct AutowareRoleNameString
+  {
+    static constexpr const char LightBulbs[] = "light_bulbs";
+  };
+
```

Comment:
Is this change intentional?
What is the reason for making this change within this PR?

Reply:
I added `AutowareRoleNameString::CrosswalkPolygon`, and moved the struct definition into the class to prevent namespace conflict.

```c++
  struct AutowareRoleNameString
  {
    static constexpr const char CrosswalkPolygon[] = "crosswalk_polygon";
  };
```

Reply:
okay, understood.